core/src/main/java/io/knotx/databridge/core/DataBridgeKnot.java

```diff
@@ -28,32 +28,30 @@
 public class DataBridgeKnot extends AbstractVerticle {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(DataBridgeKnot.class);
+  public static final String EB_ADDRESS = "databrige";
```

Comment:
Let's keep consistent with EB addresses: https://github.com/Knotx/knotx-template-engine/blob/119833d7d707e587b859d36b7c6400b270f42bb1/core/src/main/java/io/knotx/te/core/TemplateEngineKnot.java#L38

Maybe we should stay with `knotx.knot.databridge` as it grants unique Vert.x EB address?